### PR TITLE
Egress firewall supports only 50 rules

### PR DESCRIPTION
**Pull request by jboxman**
- https://issues.redhat.com/browse/RFE-155

Worth adding a note about this.

### CODE REVIEW

**Comment by jboxman** [comment]
@danwinship @juanluisvaladas 

I came across the RFE for this and wanted to add a note; Do I understand the current limitation correctly?

Thanks.

**Comment by juanluisvaladas** [comment]
@jboxman you can only create one egress network policy object per project. The object may have up to 50 rules.

Instead of: A maximum of 50 EgressNetworkPolicy objects may be defined cluster-wide at any time. 
Should be: A maximum of 1 EgressNetworkPolicy object with a maximum of 50 rules may be defined per project.

**Comment by jboxman** [comment]
@juanluisvaladas, thanks!

File: modules/nw-egressnetworkpolicy-about.adoc [hunk 1]
```diff
@@ -38,6 +38,8 @@ An egress firewall has the following limitations:
 
 * No project can have more than one EgressNetworkPolicy object.
 
+* A maximum of 1 EgressNetworkPolicy object with a maximum of 50 rules may be defined per project.
```
Comment (ahardin-rh) [inline modules/nw-egressnetworkpolicy-about.adoc:41]: s/may/can

**Pull request merged**